pinot-tools/src/test/java/org/apache/pinot/tools/streams/PinotRealtimeSourceTest.java

```diff
@@ -49,6 +49,28 @@ public void testBuilder() {
     Assert.assertEquals(realtimeSource._rateLimiter, limiter);
   }
 
+  @Test
+  public void testDefaultGuavaRateLimiter() throws Exception {
```

Comment:
Seems this does not follow `Pinot Style`

Reply:
I don't really see how this test works (almost everything is mocked). I guess we don't really need a test because the fix is self-explainable, and this test is not really guarding the fix